app/controllers/comment_controller.php

```diff
@@ -0,0 +1,55 @@
+<?php
+class CommentController extends AppController
+{
+	CONST MAX_ITEM_PER_PAGE = 10;
+
+	public function view()
+	{
+		session_start();
```

Comment:
Already moved po session checking in my layout.
https://github.com/renzosunico/BoardExercises/commit/06a9ddaac94a5583b0ead86ae868e342e33b588f